packages/stack/blockchain/src/index.js

```diff
@@ -52,14 +51,23 @@ export default class Blockchain {
         this.startedClient = clientName;
         this.events.emit("blockchain:started", clientName);
       };
-      if (clientName === constants.blockchain.vm) {
-        started();
-        return cb();
+      try {
+        const isVM = await this.events.request2('blockchain:client:vmProvider', clientName);
```

Comment:
is this really necessary? the blockchain module shoudl be agnostic to this type of thing

Reply:
TLDR: No, it's not necessary, found a new way using an action.

No, it's not, but yesterday, I tried to find a way to **not** do that long and only found **worse** solutions, so I thought this one was the best, because it's just a small request.

Now, when I tried to sleep yesterday, I couldn't stop thinking about it, and I had a flash of genius. I'm going to make the `blockchain:node:start` request and action instead, and create a new VM manager stack component. 
That way, both the VM manager and the blockchain components will register an action for the blockchain start, and they can both tell "hey, yes I have a client for that config", or "no I don,t know that client". If both of them respond false (`started: false`), then we can print the message that nothing was registered.

This way, blockchain becomes agnostic of VMs, and only the Proxy needs to know about it, but it's necessary, because it's the one that needs to connect to the node or the provider.